Huion_H950P/index.md

```diff
@@ -0,0 +1,20 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: H950P
+image: Huion_H950P
```

Comment:
This doesn't match the file name, so it won't work on the website.

Also, I forgot to check this part of the last PR, and merged it broken. Here are the requirements for the image files: `.jpg` should be lowercase, i.e. **not** `.JPG`, there should be no executable permissions on the image file, just 0644, file name should be the brand name and the model name on the photo, with underscore in between, `index.md` should contain the name of the file with the image. Could you please submit a PR fixing the previous commit, and fix this PR to match this too? Also, each addition of new images, should also add a thumbnail, which can be generated by running `./gen-thumbs` in the root directory. Plus, also install `kwalify` and run `./validate` in the root directory before submitting, and check it doesn't report any errors.'